### PR TITLE
Adding in Dockerfile and Compose file for Multi Stage Build

### PR DESCRIPTION
**Pull request by ManoMarks**
Requires latest dev build of docker to run

### CODE REVIEW

File: src/main/resources/static/home/home.controller.js [hunk 1]
```diff
@@ -10,36 +10,39 @@
...
-            loadAllUsers();
+            // loadAllUsers();
         }
 
         function loadCurrentUser() {
-            UserService.GetByUsername($rootScope.globals.currentUser.username)
-                .then(function (user) {
-                    vm.user = user;
-                });
+            // UserService.GetByUsername($rootScope.globals.currentUser.username)
```
Comment (nathanleclaire) [inline src/main/resources/static/home/home.controller.js]: Why comment out? Just remove them right? (It'll still be stored in source control if we want to get the lines back someday)

File: target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst [hunk 2]
```diff
@@ -1,17 +1,17 @@
-/Users/sophiaparafina/git/spara/mobyStore/src/main/java/com/docker/mobystore/controller/RestApiController.java
```
Comment (nathanleclaire) [inline target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst]: This file should probably not be in source control, we should probably add `target` to `.gitignore` and `git rm` it

**Comment by nathanleclaire** [comment]
The JAR should not be in source control -- not sure if you noticed you've committed that here

**Comment by ManoMarks** [comment]
Yeah I made a bunch of commits later which I accidentally added to this one. I will roll back all these commits, especially the front-end ones since Jess is working on a better front end.

**Comment by nathanleclaire** [comment]
cool thanks

**Pull request closed without merging**